src/main/java/cz/codecamp/logger/FormatterInterface.java

```diff
@@ -1,5 +1,16 @@
 package cz.codecamp.logger;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
 public interface FormatterInterface {
+
+    DateFormat df = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss");
```

Comment:
To opravdu patří sem? Imo ne: všechny formattery nemusí používat stejné formátování času.